manifest/manifestlist/manifestlist.go

```diff
@@ -10,28 +10,44 @@ import (
 	"github.com/docker/distribution/manifest"
 )
 
-// MediaTypeManifestList specifies the mediaType for manifest lists.
-const MediaTypeManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+const (
+	// MediaTypeManifestList specifies the mediaType for manifest lists.
+	MediaTypeManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+
+	// MediaTypeOCIManifestList specifies the mediaType for OCI compliant
+	// manifest lists.
+	MediaTypeOCIManifestList = "application/vnd.oci.image.manifest.list.v1+json"
+
+	// DefaultMediaTypeManifestList is the default choice of manifest list type
+	// when marshalling/serving content
+	DefaultMediaTypeManifestList = MediaTypeOCIManifestList
+)
 
 // SchemaVersion provides a pre-initialized version structure for this
 // packages version of the manifest.
 var SchemaVersion = manifest.Versioned{
 	SchemaVersion: 2,
-	MediaType:     MediaTypeManifestList,
+	MediaType:     DefaultMediaTypeManifestList,
```

Comment:
@vbatts We should really remove the media type embedded in the content itself. It makes this code much more problematic. Changing the type shouldn't change the content.